### DISCUSSION TITLE
Posibilidad de tener un ordenamiento no alfabético en los menús

### DISCUSSION THREAD

**Discussion by RioEsgueva** [FPGAwars/icestudio#847] [Ideas]
No se si ya habrá sido implementado, pero me gustaría sugerir esto: estaría bien poder añadir una propiedad de ordenamiento en el menú cuando se diseña una colección, así se puede ordenar los corazones por frecuencias aunque no sea su orden alfabético.

**Comment by cavearr** [comment]
Hello! I am reorganizing the issues, trying to give more visibility and be able to better organize and prioritize the work and my roadmap (I will publish it very soon), I am converting some issues that are **new feature requests** as **ideas** on Github **discussions**.

Thanks for understanding.
